jaxrs-api/src/main/java/jakarta/ws/rs/core/EntityPart.java

```diff
@@ -145,15 +145,19 @@ static Builder withFileName(String partAndFileName) {
      *                                  {@code getContent} methods has already been
      *                                  invoked
      * @throws IOException              if the
-     *                                  {@link jakarta.ws.rs.ext.MessageBodyReader#readFrom(Class, java.lang.reflect.Type, java.lang.annotation.Annotation[], MediaType, MultivaluedMap, InputStream)}
+     *                                  {@link jakarta.ws.rs.ext.MessageBodyReader#readFrom(Class,
+     *                                  java.lang.reflect.Type, java.lang.annotation.Annotation[], MediaType, MultivaluedMap, InputStream)}
      *                                  method throws an {@code IOException}
      * @throws WebApplicationException  if the
-     *                                  {@link jakarta.ws.rs.ext.MessageBodyReader#readFrom(Class, java.lang.reflect.Type, java.lang.annotation.Annotation[], MediaType, MultivaluedMap, InputStream)}
+     *                                  {@link jakarta.ws.rs.ext.MessageBodyReader#readFrom(Class,
+     *                                  java.lang.reflect.Type, java.lang.annotation.Annotation[], MediaType, MultivaluedMap, InputStream)}
      *                                  method throws an
      *                                  {@code WebApplicationException}
      */
+    //CHECKSTYLE:OFF - More than 3 Exceptions are desired here
```

Comment:
I am not a Checkstyle expert, but isn't it possible to just ignore a single rule?